inc/vite.php

```
@@ -180,6 +181,23 @@ public function enqueue_assets()
+     */
+    public function make_css_async($tag, $handle)
+    {
+        if (strpos($handle, 'djz-react-style-') === 0) {
+            $tag = str_replace(
+                "rel='stylesheet'",
+                "rel='stylesheet' media='print' onload=\"this.media='all'\"",
+                $tag
+            );
+        }
+        return $tag;
+    }
```

PHPMD (2.15.0)

[error] The method make_css_async is not named in camelCase. (undefined)

(CamelCaseMethodName)